src/pbrt/textures.cpp

```diff
@@ -601,7 +600,8 @@ struct : public PtexErrorHandler {
 PtexTextureBase::PtexTextureBase(const std::string &filename, ColorEncoding encoding,
                                  Float scale)
     : filename(filename), encoding(encoding), scale(scale) {
-    ptexMutex.lock();
+    std::mutex mutex;
```

Comment:
This won't work: each thread calling into this will get its own value of `mutex` and so no mutual exclusion around will happen. (What was the intent here?)

Reply:
I don't remember this change. Let me check whether it's actually needed.